apis/v1alpha2/validation/httproute_test.go

```diff
@@ -816,19 +817,43 @@ func TestValidateHTTPRouteTypeMatchesField(t *testing.T) {
 				Remove: []string{"remove"},
 			},
 		},
+		backendRefs: []gatewayv1a2.HTTPBackendRef{{
+			BackendRef: gatewayv1a2.BackendRef{
+				BackendObjectReference: gatewayv1a2.BackendObjectReference{
+					Name: gatewayv1a2.ObjectName("test"),
+					Port: ptrTo(gatewayv1b1.PortNumber(8080)),
+				},
+			},
+		}},
```

Comment:
It looks like this is repeated a lot, can you move the standard set of backendRefs to a var so there's not quite so much repetition here?

Reply:
Actually it looks like this isn't modified at all for any of the test cases, maybe just add this to the route populated in `t.Run` below?